src/redux/reducer.js

```diff
@@ -92,6 +96,7 @@ const initialState = {
     studyUuid: null,
     network: null,
     geoData: null,
+    networkModificationTreeModel: { treeElements: [] },
```

Comment:
would be better with an instance of the class ? or null like for the network ?

Reply:
Corrected to create a new instance of the class.